service/history/workflow/history_builder.go

```diff
@@ -127,9 +127,10 @@ func NewMutableHistoryBuilder(
 }
 
 func NewImmutableHistoryBuilder(
-	history []*historypb.HistoryEvent,
+	histories ...[]*historypb.HistoryEvent,
```

Comment:
`...[]*historypb.HistoryEvent,` is still an array of array, but easier for caller to make function call
so instead of 
```go
var events []*historypb.HistoryEvent
NewImmutableHistoryBuilder([][]*historypb.HistoryEvent{events})
```	

we can do
```go
```go
var events []*historypb.HistoryEvent
NewImmutableHistoryBuilder(events)
```